packages/ra-ui-materialui/src/input/ResettableTextField.js

```diff
@@ -95,7 +95,10 @@ function ResettableTextField({
             classes={restClasses}
             value={value}
             InputProps={{
-                classes: { adornedEnd: props.select ? inputAdornedEnd : null },
+                classes:
+                    props.select && variant === 'filled'
```

Comment:
I don't get the fix. Why is it needed to check the variant ?

Reply:
Because the `inputAdornedEnd` class doesn't exist in the standard variant of MUI's TextField. Unfortunately, the list of classes of all variants isn't the same, hence the warning that we have to fix.